_posts/2020-08-13-convertextract-app.md

```diff
@@ -0,0 +1,95 @@
+
+As a Student Intern on the NRC's [Indigenous Language Technology (ILT) project](https://nrc.canada.ca/en/research-development/research-collaboration/programs/canadian-indigenous-languages-technology-project), I was approached by the Kwak̓wala Corpus Collection group to help create a systemic way to streamline the quality control process for their ELAN annotation data. Having many different people with many different orthographic conventions all working on annotating Kwak̓wala language data had resulted in inconsistencies.
+So, I added mappings in the [g2p library](https://github.com/roedoejet/g2p) 
+and a parser for ELAN in the Convertextract library. Aidan Pine then added a graphical user interface to Convertextract!
+
+# How to use the new Convertextract application for 'quality control' of ELAN annotations
+
+[Convertextract](https://github.com/roedoejet/convertextract), created by Aidan Pine, is a `python` library which extracts text data and finds/replaces specific text based on arbitrary correspondences. 
+Until now, only a CLI (Command Line Interface) was supported. Using Convertextract in the CLI allowed the user to convert a file based on pre-existing Mappings in the [g2p library](https://github.com/roedoejet/g2p) or based on a custom mapping. However, the downside is that some programming knowledge is needed to use the CLI.
+The latest update now includes a GUI (Graphical User Interface) in the form of an application (for Mac **only**). The app makes Convertextract more accessible for non-programmers.
+
+### 1. G2P mapping
+G2P mappings (i.e. arbitrary conversions between an input and an output) can be created manually as a `.csv` or `.json` file, or with the [g2p studio app](g2p-studio.herokuapp.com).
+Many languages are already supported. See the [g2p repository](https://github.com/roedoejet/g2p) on Github for detailed documentation on the G2P library.
+
+### 2. Language data
+
+Your language data must be in one of the supported file formats. The most recent addition is `.eaf` files, which allows ELAN annotations to be parsed and converted. For a full list of supported file types click [here](https://github.com/roedoejet/convertextract).
+
+### 3. Convertextract application
+
+#### Installation
+To download the app: [https://github.com/roedoejet/convertextract/releases](https://github.com/roedoejet/convertextract/releases).
+
+In your downloads folder, find the .zip file and double click on it to unzip.
+- Downloads>**convertextract**
+
+Right-click on the application in the dist folder and select *Open*.
+- Downloads>convertextract>dist>**Convertextract**
+
+*Note: If you try to double click to open the app, you will get a security message. Right-clicking to open will allow you to override the security message.*
+
+#### Using the app
+
+This is what the app looks like when you open it.
+
+![gui]([https://raw.githubusercontent.com/roedoejet/mothertongues-blog/assets/images/gui.png])
```

Comment:
```suggestion
![Convertextract GUI]({{ 'assets/images/why-cree-syllabics--mayor-and-okimaw.jpg' | relative_url }})
```

Honestly, we (read: Aidan and I) could make embedding images easier. I'll look into this!